_includes/footer.html

```diff
@@ -1,17 +1,17 @@
+<!--TODO this about how we manage both sets of assets UIkit and cloud -->
 <footer role="contentinfo">
   <div class="wrapper">
     <div class="footer-logo">
-      <img src="https://www.gov.au/alpha/assets/coatofarms-01.svg" alt="Logo image" onerror="this.src='https://www.gov.au/alpha/assets/coatofarms-01.png'; this.onerror=null;">
+      <img src="https://raw.githubusercontent.com/AusDTO/mkdocs-gov-au-theme/master/gov_au_theme/assets/images/coat-of-arms.png" alt="Australian Government Coat of Arms">
```

Comment:
Would a better solution be to add this image to this Jekyll site _assets and reference it as a local asset?